src/main/resources/graphql/.graphqlconfig

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Tror kanskje jeg ville droppet denne filen. Hvis du ønsker å legge til dokumentasjon for hvordan man kan teste PDL direkte så passer det kanskje bedre i en README-fil

Reply:
Ok. Jeg skal legge den til.